### PR TITLE
Pi tft43  mount

### PR DESCRIPTION
**Pull request by maramauphoto**
* [✓ ] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [ ✓] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [ ✓] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [✓ ] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [ ✓] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [ ✓] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [✓ ] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [✓ ] Submitted STL files are oriented and scaled properly for printing.

#### Which mods/configurations/profiles are added by this PR?
a screen mount for the BTT pitft43 screen

#### How was it tested? 
printed and fitted

#### Any background context you want to provide?
i needed a mount for the screen i previously purchased

### CODE REVIEW

File: printer_mods/README.md [hunk 1]
```diff
@@ -174,6 +174,7 @@ like so:
 |Majarspeed| [Hartk board mount galileo](./Majarspeed/Hartk_Mount_Galileo) | It hangs stuff on your tool | :x: | :x: | :heavy_check_mark: | :x: |
 |madfabrications |[Vertical PSU Mount](./madfabrications/Vertical_PSU_Mount) |Mounts the PSU vertically on the DIN rails|:x: |:x: |:heavy_check_mark:|:x:|
 |madros |[V0 OLED-skirt](./madros/V0_skirt_OLED) |V0 skirt with 0.96" OLED|:heavy_check_mark: |:x: |:x: |:x:|
+|Maramao| [BTT piTFT43 Mount](.\maramao\STL\BTT_piTFT43_mount) | Mount for the BTT piTFT43 screen | :x: | :x: | :heavy_check_mark: | :grey_question: |
```
Comment (KiloQubit) [inline printer_mods/README.md:177]: This link is broken - the slashes are backwards and the directory is incorrect.  If you move your mod to `printer_mods/maramao/BTT_piTFT43_mount/` as suggested in my other comment, then your link here would be

    [BTT piTFT43 Mount](./maramao/BTT_piTFT43_mount)

Also please add a question mark or an X in the last column (Trident) of the compatibility matrix.

File: printer_mods/maramao/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# BTT piTFT43 v2 Mount
+
+this is a quick remix based on the work of alanho [BTT_PITFT50_v2_Mount] (https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/alanho/BTT_PITFT50_v2_Mount)
```
Comment (KiloQubit) [inline printer_mods/maramao/README.md:3]: Please remove the space between [] and (), and change the link to relative.  It should be

    [BTT PITFT50 v2 Mount](../../alanho/BTT_PITFT50_v2_Mount)

Comment (KiloQubit) [inline printer_mods/maramao/README.md:3]: Capitalize `t` in `This` at the start of the sentence.

File: printer_mods/maramao/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# BTT piTFT43 v2 Mount
+
+this is a quick remix based on the work of alanho [BTT_PITFT50_v2_Mount] (https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/alanho/BTT_PITFT50_v2_Mount) 
+to fit on the screen mount of [sttts's Waveshare 5.5inch mount] (https://github.com/VoronDesign/VoronUsers/blob/master/printer_mods/sttts/Waveshare-5.5-inch-HDMI-AMOLED)
```
Comment (KiloQubit) [inline printer_mods/maramao/README.md:4]: Remove space between [] and () and change link to relative:

    [sttts's Waveshare 5.5inch mount](../../sttts/Waveshare-5.5-inch-HDMI-AMOLED)

File: printer_mods/maramao/README.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+to use the BTT piTFT43 
+
+## BOM
+4x M3x6mm
```
Comment (KiloQubit) [inline printer_mods/maramao/README.md:8]: Make this a list so it doesn't show all on one line:

```
- 4x M3x6mm
- 4x M2.5x4mm (included in the BTT PITFT50 v2)
- 500mm ~ 600mm Ribbon Flat Cable for Raspberry Pi Camera (depends on the position of the Raspberry Pi. The included one is too short)
```

File: printer_mods/maramao/README.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+4x M2.5x4mm (included in the BTT PITFT50 v2)
+500mm ~ 600mm Ribbon Flat Cable for Raspberry Pi Camera (depends on the position of the Raspberry Pi. The included one is too short)
+
+## Instrucions
```
Comment (KiloQubit) [inline printer_mods/maramao/README.md:12]: Replace `Instrucions` with `Instructions`

File: printer_mods/maramao/README.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+Use the mount in [sttts's mods] (https://github.com/VoronDesign/VoronUsers/blob/master/printer_mods/sttts/Waveshare-5.5-inch-HDMI-AMOLED/STLs/Voron-2.4-Mount-Generisch-v6.stl) to mount this to the frame.
```
Comment (KiloQubit) [inline printer_mods/maramao/README.md:13]: Remove space between [] and () and change to relative link:

    [sttts's mods](../../sttts/Waveshare-5.5-inch-HDMI-AMOLED/STLs/Voron-2.4-Mount-Generisch-v6.stl)

File: printer_mods/maramao/README.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+
```
Comment (KiloQubit) [inline printer_mods/maramao/README.md:14]: Embed the pictures you have in the README with something like this

```
![Mount side view](Pictures/1.jpg)
![Mount front view](Pictures/2.jpg)
![Display](Pictures/3.jpg)
```

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

Please create a directory under your username and move the mod there, e.g. put your mod in `printer_mods/maramao/BTT_piTFT43_mount/` instead of directly in `printer_mods/maramao/` where it is now.

You have two STLs there that seem identical except chamfers on the underside of the holes - `piTFT43_mount.stl` (good) and `BTT_piTFT43_mount.stl` (missing chamfer on hole underside) - did you mean to only include `piTFT43_mount.stl`?

**Comment by maramauphoto** [comment]
> Thanks for your mod!
> 
> Please create a directory under your username and move the mod there, e.g. put your mod in `printer_mods/maramao/BTT_piTFT43_mount/` instead of directly in `printer_mods/maramao/` where it is now.
> 
> You have two STLs there that seem identical except chamfers on the underside of the holes - `piTFT43_mount.stl` (good) and `BTT_piTFT43_mount.stl` (missing chamfer on hole underside) - did you mean to only include `piTFT43_mount.stl`?

No, I meant to include only the BTT_piTFT43_mount.stl

File: printer_mods/maramao/README.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+This is a quick remix based on the work of alanho [BTT_PITFT50_v2_Mount](../../alanho/BTT_PITFT50_v2_Mount) 
+to fit on the screen mount of [sttts's Waveshare 5.5inch mount](../../sttts/Waveshare-5.5-inch-HDMI-AMOLED)
+
+## BOM
+- 4x M3x6mm
+- 4x M2.5x4mm (included in the BTT PITFT50 v2)
+- 500mm ~ 600mm Ribbon Flat Cable for Raspberry Pi Camera (depends on the position of the Raspberry Pi. The included one is too short)
+
+## Instructions
+Use the mount in [sttts's mods] (../../sttts/Waveshare-5.5-inch-HDMI-AMOLED/STLs/Voron-2.4-Mount-Generisch-v6.stl) to mount this to the frame.
```
Comment (KiloQubit) [inline printer_mods/maramao/README.md:12]: This still needs space removed between `[]` and `()`

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for the update!

- You still don't have a directory for your mod under the username.  Please create `printer_mods/mramao/BTT_piTFT43_mount/` directory and move your mod files in there
- you have unprintable overhangs on the bottom of two holes, you need to chamfer the holes to avoid that
Screen Shot 2021-09-20 at 17 49 58

- Your STL file is corrupted

**Comment by bdbell** [comment]
Will the last two items be updated so the PR can be accepted?

**Comment by cbc02009** [comment]
Closing due to inactivity.

**Pull request closed without merging**